owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/shares/RemoteShare.kt

```diff
@@ -180,9 +180,8 @@ class RemoteShare : Parcelable, Serializable {
         const val FEDERATED_PERMISSIONS_FOR_FOLDER_AFTER_OC9 =
             FEDERATED_PERMISSIONS_FOR_FOLDER_UP_TO_OC9 + SHARE_PERMISSION_FLAG
 
-
-        const val INIT_EXPIRATION_DATE_IN_MILLIS : Long = 0
```

Comment:
according to [Kotlin style guide](https://developer.android.com/kotlin/style-guide), your code were Okay but now it isn't.

Reply:
I'd say the whitespace before a colon ( : ) is "only used with a class declaration for specifying a base class or interfaces", copied-pasted from Kotlin style guide

Reply:
Oops! You're right. I just take a quick look to examples without reading everything 😅 

Actually, I doubted  when I saw it 😄